_posts/2020-12-10-share-your-knowledge.markdown

```diff
@@ -0,0 +1,59 @@
+---
+layout: epic
+title: "Share You a Knowledge for Great Good!"
+date: 2020-12-10
+categories: [teams, culture, people]
+author: ash
+---
+
+Sharing knowledge! What a concept! [In my recent blog post](/blog/becoming-mobile-first-at-artsy), I discussed "Knowledge Share" meetings (also known simply as "Knowledge Shares", or abbreviated "KS") and I want to dive deeper into them today. Last time, I described them as follows:
+
+> Knowledge Shares are a **structured** time to facilitate **unstructured learning**. Anyone can bring a topic to Knowledge Share, from a ticket that they're stuck on to an idea they have to a neat trick they recently learned.
+
+These meetings were really instrumental in ramping up the Mobile Experience team, but their history goes back a bit further. Today, we're going to discuss the origins of Knowledge Shares at Artsy, how they've evolved, the value they provide us as engineers, and how I'd recommend you adopt them on your team.
+
+Let's go!
+
+<!-- more -->
+
+To my recollection, the Auctions team was the first at Artsy to start doing "Knowledge Share" meetings. It was late 2016, I think. We were a small team of about five engineers, but were spread across many different front-end apps and back-end microservices. To make sure Artsy's nascent auction business was a success, every engineer on the team had to have a working understanding of every one of our codebases. The team's lead, Alan, suggested that devote an hour each week to sharing knowledge so no one person would become a silo.
+
+The idea is simple enough: spend time working on something together. Kind of like pair programming, but with a slightly bigger group. Maybe there would be a ticket someone had that we would start together. Or maybe there had been a production incident that we wanted to investigate together. Maybe someone had an idea for a new technical approach to some problem and they wanted early feedback. 
+
+I think this forum worked well for us because our team was small and had a high level of interpersonal trust. Leading a five-person pairing session could be really intimidating, and later I'll discuss how I've since structured the Knowledge Shares that I lead to make everyone feel welcome contributing.
```

Comment:
```suggestion
I think this forum worked well for us because our team was small and had a high level of interpersonal trust. Leading a five-person mobbing session could be really intimidating, and later I'll discuss how I've since structured the Knowledge Shares that I lead to make everyone feel welcome contributing.
```

Non-blocking: I don't know if it's truly "mob programming" by the book, but that feels more accurate than "pairing", given there are 5 people.